docsrc/conf.py

```diff
@@ -277,4 +279,45 @@ def setup(app):
 
 numpydoc_xref_ignore = {'type', 'optional', 'default', 'shape', 'fitted', 'instance',
                         'cluster', 'of', 'or', 'if', 'using', 'otherwise', 'required',
-                        'from'}
\ No newline at end of file
+                        'from'}
+
+
+# This is processed by Jinja2 and inserted before each notebook
+nbsphinx_prolog = r"""
+{% set docname = env.doc2path(env.docname, base='doc') %}
```

Comment:
shouldn't this be `docsrc` (see Line 296)? Where is it being generated from? See [here](https://github.com/pysal/spaghetti/blob/c8398fcb13b442e30f1954ebe4d6a2d3b70f7200/docsrc/conf.py#L272-L287) in `spaghetti` for example.

Reply:
great catch!